packages/gluestick/src/renderer/main.js

```diff
@@ -136,24 +120,11 @@ module.exports = function startRenderer({ config, logger }: Context) {
 
     readAssets(assetsFilename)
       .then((assets: Object): Promise<void> => {
-        return middleware(
-          { config, logger },
-          req,
-          res,
-          { entries, entriesConfig, entriesPlugins: runtimePlugins },
-          { EntryWrapper, BodyWrapper },
-          { assets, loadjsConfig: applicationConfig.loadjs || {} },
-          {
-            reduxMiddlewares,
-            thunkMiddleware,
-            reduxEnhancers,
-            envVariables,
-            httpClient: applicationConfig.httpClient || {},
-            entryWrapperConfig: {},
-          },
-          { hooks, hooksHelper: hooksHelper.call },
+        return middleware({ config, logger }, req, res, {
+          assets,
```

Comment:
assets: comes from a cached file. `webpack-flush-chunks` changes this.
hooks: requires serverPlugins to create, which is completely static _except_ for `logger`. Will be imported directly after `logger` becomes a singleton.
serverPlugins: same, reliant on `logger`.